ParkenDD/Colors.swift

```diff
@@ -65,6 +65,7 @@ extension UIColor {
 					red   = CGFloat((hexValue & 0xF00) >> 8)       / 15.0
 					green = CGFloat((hexValue & 0x0F0) >> 4)       / 15.0
 					blue  = CGFloat(hexValue & 0x00F)              / 15.0
+                    alpha = CGFloat(hexValue & 0x000F)             / 15.0
```

Comment:
Make sure your indentation is the same as the project. I believe it's set to 4/4 spaces. The code you submitted may look good on Xcode but it's going to be distracting when reviewing it.

Reply:
Fixed indentation and submitted a commit with the changes.